movo_7dof_moveit_config/launch/move_group.launch

```diff
@@ -50,7 +50,7 @@
 
     <!-- MoveGroup capabilities to load -->
     <param name="capabilities" value="move_group/MoveGroupCartesianPathService
-				      move_group/MoveGroupExecuteService
+				      move_group/MoveGroupExecuteTrajectoryAction
```

Comment:
is this a soln u found online?

Reply:
Yeah I found that MoveGroupExecuteService is deprecated and replaced with "TrajectoryAction"
https://github.com/ros-industrial/universal_robot/issues/413 
https://github.com/ros-planning/moveit/issues/201